hawc/apps/udf/forms.py

```diff
@@ -23,19 +35,38 @@ def __init__(self, *args, **kwargs):
 
     class Meta:
         model = UserDefinedForm
-        exclude = ("parent_form", "creator", "created", "last_updated")
+        fields = ("name", "description", "schema", "editors", "deprecated")
         widgets = {
             "editors": AutocompleteSelectMultipleWidget(UserAutocomplete),
         }
 
     @property
     def helper(self):
         self.fields["description"].widget.attrs["rows"] = 3
-        cancel_url = reverse("portal")  # TODO: replace temp cancel url
-        helper = BaseFormHelper(
-            self,
-            legend_text="Create a custom data extraction form",
-            cancel_url=cancel_url,
-            submit_text="Create Form",
+        cancel_url = reverse("udf:udf_list")
+        form_actions = [
+            cfl.Submit("save", "Save"),
+            cfl.HTML(f'<a role="button" class="btn btn-light" href="{cancel_url}">Cancel</a>'),
+        ]
+        legend_text = "Update a custom form" if self.instance.id else "Create a custom form"
+        helper = BaseFormHelper(self)
+        helper.layout = cfl.Layout(
```

Comment:
this looks very unstyled - that's by design right? Are you thinking we'll come back later and add UI polish + tests?  Makes sense to me, just want to confirm that's what you're thinking...

Reply:
Yup that's what I had in mind